lib/Widget/dashboard/pre_operation/preDashboardDetail.dart

```diff
@@ -50,6 +50,13 @@ class _PreDashboardDetailState extends State<PreDashboardDetail> {
                       onPressed: () {
                         print('$heading : ${widget.hn}');
                         setState(() {});
+                        // Navigator.push(
+                        //     context,
+                        //     MaterialPageRoute(
+                        //       builder: (context) => ADLForm(
+                        //         hn: widget.hn,
+                        //       ),
+                        //     ));
```

Comment:
Is this code still in use?

Reply:
Just in case of debug. will be used in the next PR. Then, remove.